src/main/java/org/opentripplanner/routing/edgetype/TripPattern.java

```diff
@@ -100,16 +101,12 @@
      */
     public String code;
 
-    /* The vertices in the Graph that correspond to each Stop in this pattern. */
-    public final TransitStop[] stopVertices; // these are not unique to this pattern, can be shared. FIXME they appear to be all null. are they even used?
-    public final PatternDepartVertex[] departVertices;
-    public final PatternArriveVertex[] arriveVertices;
-
-    /* The Edges in the graph that correspond to each Stop in this pattern. */
-    public final TransitBoardAlight[]  boardEdges;
-    public final TransitBoardAlight[]  alightEdges;
-    public final PatternHop[]          hopEdges;
-    public final PatternDwell[]        dwellEdges;
+    /**
+     * The vertices in the Graph that correspond to each Stop in this pattern.
+     * Note: these are not unique to this pattern, and could be shared in the stop.
+     * FIXME they appear to be all null. are they even used?
+     */
+    public final TransitStop[] stopVertices;
```

Comment:
It is not used, just initalized in PatternHopFactory. I will add a TODO OTP2 with comments.